ffi/diplomat/js/examples/node/build.sh

```diff
@@ -29,7 +29,7 @@ if ! test -f "full-data-cached.postcard"; then
         --all-locales \
         --all-keys \
         --cldr-tag 41.0.0 \
-        --icuexport-tag icu4x/2022-08-17/71.x \
+        --icuexport-tag release-72-1 \
```

Comment:
Thought: Maybe we should keep this one on the old data version? But I guess we want the WASM demo to be using the latest data. https://github.com/unicode-org/icu4x/issues/2782#issuecomment-1300991060